src/index.js

```diff
@@ -4,7 +4,24 @@ import path from 'path';
 // markdown parser and html converter
 import mdit from 'markdown-it';
 import mdcomment from 'markdown-it-inline-comments';
-const md = mdit().use(mdcomment);
+import hljs from "highlight.js";
+
+var md = mdit({
+	highlight: function (str, lang) {
+		if (lang && hljs.getLanguage(lang)) {
+			try {
+				return hljs.highlight(str, { language: lang }).value;
+			} catch (__) {}
+		}
+
+		try {
+			return hljs.highlightAuto(str).value;
+		} catch (__) {}
+
+		return ''; // use external default escaping
```

Comment:
I think printing an error if the syntax highlighting failed is probably a good idea.